Python/Text Editor/README.md

```diff
@@ -0,0 +1,11 @@
+## Feature Type:
+
+- It is a simple Text Editor 
+- Fully Written in Python
+- Supports Windows, MacOS, Linux(Testing Not Done)
+- It can change font to and font size also
+- It can be good upon more contribution
+
+## How to Run
+
+- Just Run This Python Program and see the Magic.
```

Comment:
@Avik-creator follow the rules which are mentioned in the #43 . Readme file is not following.